images/win/scripts/ImageHelpers/MarkdownHelpers.ps1

```diff
@@ -1,5 +1,5 @@
 function Add-ContentToMarkdown {
-    [CmdletBinding()]
+    
```

Comment:
Is this supposed to be here? I just noticed I am deleting the binding? but I never touched this file myself?